tests/helpers.py

```diff
@@ -380,21 +388,34 @@ def delete_cephblockpool():
         kind=constants.CEPHBLOCKPOOL,
         namespace=defaults.ROOK_CLUSTER_NAMESPACE
     )
-    pool_list = get_cephblockpool_name()
-    for item in pool_list:
-        logger.info(f"Deleting CephBlockPool with name {item}")
-        assert POOL.delete(resource_name=item)
+    logger.info(f"Deleting CephBlockPool with name {cbp_name}")
+    assert POOL.delete(resource_name=cbp_name)
     return True
 
 
-def create_cephfilesystem():
+def create_cephfilesystem(override_fs=True):
     """
     Function for deploying CephFileSystem (MDS)
 
+    Args:
+        override_fs (bool): If true it will delete old CFS and recreate new one, If False will skip creation of CFS
+
     Returns:
         bool: True if CephFileSystem creates successful
     """
     fs_data = copy.deepcopy(defaults.CEPHFILESYSTEM_DICT)
+    if override_fs:
+        logger.info("Deleting CephFileSystem if exists")
+        delete_all_cephfilesystem()
```

Comment:
Not good,

You will delete the base FS

Reply:
override_fs will decide delete or not if user set override_fs as false it will not delete

Reply:
by default it will no delete an existing one because override_fs is set to False if user need to create new mds he/she can pass override_fs as True